app/models/qernel/graph.rb

```diff
@@ -402,6 +402,17 @@ def graph_query
     @graph_query ||= GraphApi.new(self)
   end
 
+  def initializer_inputs
+    if init = area.init
+      init.
+        map { |key, val| [Input.fetch(key), val] }.
+        sort_by { |input, _| input.priority }.
+        reverse
```

Comment:
I'm a bit of a fan of splitting these things up in several methods like:

````ruby
def initializer_inputs
  decorated_inputs.sort_by { |input, _| input.priority }.reverse
end

private

def decorated_inputs
  (area.init || {}).map do |key, val|
    [Input.fetch(key), val]
  end
end
````

... or something like that.